hr_attendance_rfid/security/hr_attendance_rfid.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="group_hr_attendance_rfid" model="res.groups">
+        <field name="name">RFID Attendance</field>
+        <field
+            name="category_id"
+            ref="base.module_category_human_resources_attendances"
```

Comment:
Putting into this category breaks the dropdown, so it should be in another like technical one.